### PR TITLE
DOC: use mathjax on sphinx - #15469 Exponentially Weighed Windows pages now shows formulas

### LINKED ISSUE #15469: DOC: Exponentially Weighted Windows page not showing formulas
The Exponentially Weighted Windows page is not rendering the formulas in LaTeX format.
http://pandas.pydata.org/pandas-docs/stable/computation.html#exponentially-weighted-windows

### PR DESCRIPTION
**Pull request by cdemulde**
closes #15469

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -46,8 +46,9 @@
               'ipython_sphinxext.ipython_console_highlighting',
               'sphinx.ext.intersphinx',
               'sphinx.ext.coverage',
-              'sphinx.ext.pngmath',
-              'sphinx.ext.ifconfig',
+              #'sphinx.ext.pngmath',
```
Comment (jreback) [inline doc/source/conf.py]: I did see ``pngmath`` was deprecated. Is this needed for anything else though? can you build locally and give a check? (you can just ping here if ok)

**Comment by jorisvandenbossche** [comment]
So indeed pngmath is deprecated. We have to choose between `sphinx.ext.imgmath` or `sphinx.ext.mathjax` as described below. I would personally go with mathjax (you don't need latex install for building the docs, but it renders only if you are online, but for html docs I think this is OK)

http://www.sphinx-doc.org/en/stable/ext/math.html

File: doc/source/conf.py [hunk 2]
```diff
@@ -46,8 +46,8 @@
               'ipython_sphinxext.ipython_console_highlighting',
               'sphinx.ext.intersphinx',
               'sphinx.ext.coverage',
-              'sphinx.ext.pngmath',
-              'sphinx.ext.ifconfig',
+              'sphinx.ext.mathjax',
```
Comment (jreback) [inline doc/source/conf.py]: well have to remove the whitespace on merge

**Comment by jreback** [comment]
lgtm (minor comment)

**Comment by jreback** [comment]
@jorisvandenbossche ok on merging this?

**Comment by jorisvandenbossche** [comment]
Yep. Fixed the whitespace before merging.

@cdemulde Thanks! 
(btw, you should set your editor settings to not introduce tabs, but 4 spaces instead of tab)

**Comment by jreback** [comment]
https://travis-ci.org/pandas-dev/pandas/jobs/209368323

I restarted this twice and failed at the same point :<

**Comment by jorisvandenbossche** [comment]
The problem with the parallel building is that it just says "waiting for workers" instead of giving output, so difficult to see what is going on. In any case, this worked locally. Will try with more verbose output.

**Comment by jreback** [comment]
maybe add ``-v`` for now? and certainly take out ``-j 2`` as not helping :<

**Pull request merged**